torch/_inductor/compile_fx.py

```diff
@@ -243,7 +243,10 @@ def compile_fx_inner(
     # correct we will need to fix.
 
     with V.set_fake_mode(fake_mode):
-        post_grad_passes(gm)
+        # has some issues with memory in training
+        locality_reorder = is_inference and config.reordering
```

Comment:
Perhaps pass is_inference into `post_grad_passes`

Reply:
will do in follow up since benchmarking isnt in a great state currently